spec/requests/evidence_hub_spec.rb

```diff
@@ -1,31 +1,64 @@
 RSpec.describe 'Evidence Hub Summaries', type: :request do
   describe 'GET /evidence_hub' do
     context 'arriving on the Evidence Hub page' do
-      before { get '/en/evidence_hub' }
-
       it 'responds successful' do
+        get '/en/evidence_hub'
         expect(response).to be_ok
       end
+
+      it 'sends the evidence hub document types' do
+        expect(Mas::Cms::Document).to receive(:all).with(
+          params: { document_type: ['Insight'] }
+        ).and_return([])
```

Comment:
how come this should be returning `[]` ?

Reply:
Mas::Cms::Document returns an array and the test needs to return something for the next step in the index method, otherwise the test fails with `undefined method `map' for nil:NilClass`